### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by eikeon** [117/alpaca#49]
**Description**
Trying to upgrade to 4.2.2 I'm getting the following compilation error:

```
node_modules/@master-chief/alpaca/@types/stream.d.ts:30:5 - error TS2416: Property 'on' in type 'AlpacaStream' is not assignable to the same property in base type 'EventEmitter<string | symbol, any>'.
  Type '<U extends "open" | "close" | "error" | "message" | "authenticated" | "trade" | "trade_updates" | "account_updates" | "quote" | "aggregate_minute">(event: U, listener: AlpacaStreamEvents[U]) => this' is not assignable to type '<T extends string | symbol>(event: T, fn: (...args: any[]) => void, context?: any) => this'.
    Types of parameters 'event' and 'event' are incompatible.
      Type 'T' is not assignable to type '"open" | "close" | "error" | "message" | "authenticated" | "trade" | "trade_updates" | "account_updates" | "quote" | "aggregate_minute"'.
        Type 'string | symbol' is not assignable to type '"open" | "close" | "error" | "message" | "authenticated" | "trade" | "trade_updates" | "account_updates" | "quote" | "aggregate_minute"'.
          Type 'string' is not assignable to type '"open" | "close" | "error" | "message" | "authenticated" | "trade" | "trade_updates" | "account_updates" | "quote" | "aggregate_minute"'.
            Type 'T' is not assignable to type '"aggregate_minute"'.
              Type 'string | symbol' is not assignable to type '"aggregate_minute"'.
                Type 'string' is not assignable to type '"aggregate_minute"'.

30     on<U extends keyof AlpacaStreamEvents>(event: U, listener: AlpacaStreamEvents[U]): this;
```

Any ideas?

I was able to build against `4.0.5` and was trying to upgrade from that to `4.2.2`

**Expected**

Would like to be able to upgrade ;)

**Reproduction**

I'll keep digging a bit and try to trim down a way to reproduce if we can't figure it out -- hoping you might have a clue

**Logs**
If applicable, add logs to help explain the bug.

**Additional**

**Comment by 117** [comment]
I know exactly what caused it one sec.

**Comment by 117** [comment]
Try the latest `4.2.3` and let me know if this still occurs. Fixed in 0564781c6f86e18e08d4339a99215a8871c92965.

**Comment by eikeon** [comment]
That did it; thank you!
